test/DynamoCoreWpfTests/VisualizationManagerUITests.cs

```diff
@@ -698,43 +639,122 @@ private void Open(string relativePath)
         }
     }
 
-    internal static class Watch3DViewExtensions
```

Comment:
I converted all of these extension methods to looking at render packages instead of the actual collections in `Watch3DView`, because those collections are no longer available if we null them after rendering. @ramramps, again, this might conflict with the fixes you are presently creating for the `VisualizationManagerUITests` on your branch.